main.py

```diff
@@ -93,6 +96,65 @@ def __init__(self, connection) -> None:
         self.MappingThread.sig_print.connect(self.printToConsole)
         self.MappingThread.start()
 
+    def setup_buttons(self):
+        """
+
+        """
+
+        grid_width = math.ceil(math.sqrt(len(self.rocket.buttons)))
+
+        # Trying to replicate PyQt's generated code from a .ui file as closely as possible. Hence the execs and custom
+        # button object names.
+        row = 0
+        col = 0
+        for button in self.rocket.buttons.keys():
+            exec(f"self.{button}Button = QtWidgets.QPushButton(self.centralwidget)")
+            sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Minimum)
+            sizePolicy.setHorizontalStretch(0)
+            sizePolicy.setVerticalStretch(0)
+            exec(f"sizePolicy.setHeightForWidth(self.{button}Button.sizePolicy().hasHeightForWidth())")
+            exec(f"self.{button}Button.setSizePolicy(sizePolicy)")
+            font = QtGui.QFont()
+            font.setPointSize(35)
+            font.setKerning(True)
+            exec(f"self.{button}Button.setFont(font)")
+            exec(f"self.{button}Button.setObjectName('{button}Button')")
+            exec(f"self.gridLayout_5.addWidget(self.{button}Button, {row}, {col}, 1, 1)")
+            if col + 1 < grid_width:
+                col += 1
+            else:
+                col = 0
+                row += 1
+        # A .py file created from a .ui file will have the labels all defined at the end, for some reason. Two for loops
+        # are being used to be consistent with the PyQt5 conventions.
+        for button in self.rocket.buttons.keys():
+            exec(f"self.{button}Button.setText(QtCore.QCoreApplication.translate('MainWindow', '{button}'))")
+
+        for button, command in self.rocket.buttons.items():
+            exec(f"self.{button}Button.clicked.connect(lambda _: self.sendCommand('{command}'))", {"self": self})
+
+    def setup_labels(self):
+        row = 0
+        for label in self.rocket.labels:
+            name = label.name
+            exec(f"self.{name}Text = QtWidgets.QLabel(self.centralwidget)")
```

Comment:
```suggestion
            setattr(self, name + "Text", QtWidgets.QLabel(self.centralwidget))
```
I think we can use `setattr` here.